.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -63,6 +63,14 @@ on:
           - conda
         required: false
 
+      workflow-root:
```

Comment:
"Workflow" is an overloaded term in this context (GitHub workflow vs. pathogen workflow). What about a simpler name like `path` or `directory`? That would look like:

```yaml
jobs:
  pathogen-ci:
    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
    with:
      path: phylogenetic/
```

Reply:
Hmm, I'm not sure the simpler path is as clear. Yes there's ambiguity in `workflow-root` - but `path` is even more ambiguous.

Reply:
Ok, also fine for this to be non-blocking. It works, I don't have any better suggestions, and this is something we can change easily if it's only used internally.

Reply:
`build-dir`, i.e. "the path for `nextstrain build`"

Reply:
I'd be happy with `build-dir` or `build-root`